linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_suid_privilege_function/rule.yml

```diff
@@ -44,7 +44,7 @@ references:
     cis@ubuntu2004: 4.1.15
     disa: CCI-001814,CCI-001882,CCI-001889,CCI-001880,CCI-001881,CCI-001878,CCI-001879,CCI-001875,CCI-001877,CCI-001914,CCI-002234
     nist: CM-5(1),AU-7(a),AU-7(b),AU-8(b),AU-12(3),AC-6(9)
-    srg: SRG-OS-000326-GPOS-00126,SRG-OS-000327-GPOS-00127,SRG-OS-000337-GPOS-00129,SRG-OS-000348-GPOS-00136,SRG-OS-000349-GPOS-00137,SRG-OS-000350-GPOS-00138,SRG-OS-000351-GPOS-00139,SRG-OS-000352-GPOS-00140,SRG-OS-000353-GPOS-00141,SRG-OS-000354-GPOS-00142,SRG-OS-000358-GPOS-00145,SRG-OS-000359-GPOS-00146,SRG-OS-000365-GPOS-00152
```

Comment:
I guess all these SRGs come from the SLE15 STIG, for example: https://www.stigviewer.com/stig/suse_linux_enterprise_server_15/2021-06-14/finding/V-234963

It's really difficult to determine which one is correct without going really deep in understanding the SRGs.

Reply:
The question is, which direction should we take? Keep consistent with RHEL STIG or try to cover everything

Reply:
So it seems that for RHEL 8 we only have [SRG-OS-000326-GPOS-00126 and SRG-OS-000327-GPOS-00127](https://www.stigviewer.com/stig/red_hat_enterprise_linux_8/2020-11-25/finding/V-230386). So do we want go with all the ones that SUSE has on RHEL 9 or go with what we had on RHEL 8?

Reply:
these are tricky questions... we probably can start with RHEL8 only and expand in the future if needed